Rubberduck.Resources/Inspections/InspectionInfo.resx

```diff
@@ -343,4 +343,7 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="AssignmentNotUsedInspection" xml:space="preserve">
     <value>An assignment is immediately overridden by another assignment or is never referenced.</value>
   </data>
+  <data name="UnderscoreInPublicClassModuleMemberInspection" xml:space="preserve">
+    <value>A class module with public members with underscores cannot implement interfaces that do not expose said member.</value>
```

Comment:
Wording is a bit awkward. How flipping it around, and adding a bit more explanation:

> A class module that contains members with underscores cannot be implemented by other classes. The underscore is used as a separator between the interface/object name and the implemented member name: having an underscore in the member name confuses the compiler, which then refuses to compile the project. Avoid underscores in public member names by following a 'PascalCase' naming convention.